src/classes/Pop.ts

```diff
@@ -145,6 +143,20 @@ class Pop {
         const bestPath = bisectLargestAngles(gap);
         return bestPath;
     }
+
+    /**
+     * Pops will wander in random directions while no one is in their FOV
+     */
+    wander(pop: Pop, x: number, y: number, duration: number, deltaT: number) {
```

Comment:
I think we shouldn't need this extra `pop` parameter, when `this` should suffice.

Reply:
Also not sure if `duration` really needs to be a parameter here.